### PR TITLE
SLING-9656 gracefully handle scenarios where the AggregateFeaturesMojo gets invoked more than once during the build

### PR DESCRIPTION
**Pull request by enapps-enorman**
These changes appear to fix the problem locally, but I would appreciate a review to verify it doesn't cause any adverse side effects.

### CODE REVIEW

File: src/main/java/org/apache/sling/feature/maven/mojos/AggregateFeaturesMojo.java [hunk 1]
```diff
@@ -65,6 +68,15 @@
     @Override
     public void execute() throws MojoExecutionException {
         checkPreconditions();
+
```
Comment (cziegeler) [inline src/main/java/org/apache/sling/feature/maven/mojos/AggregateFeaturesMojo.java:71]: I think this doesn't work - it's totally legitimate to use the aggregate mojo more than once in a project, especially when you use profiles with different aggregations

Comment (enapps-enorman) [reply src/main/java/org/apache/sling/feature/maven/mojos/AggregateFeaturesMojo.java:68]: Thanks for reviewing and I see your point.  I left a comment on the JIRA ticket for your consideration.

**Review by cziegeler — APPROVED** [review]
Thanks, lgtm

**Comment by bosschaert** [comment]
👍

**Pull request merged**